ocs_ci/utility/utils.py

```diff
@@ -632,6 +633,11 @@ def exec_cmd(
             and b"command terminated with exit code 1" in completed_process.stderr
         ):
             log.info(f"No results found for grep command: {masked_cmd}")
+        elif b"Error from server (AlreadyExists)" in completed_process.stderr:
```

Comment:
Im not sure if its right to take care of this in the `exec_cmd` when you can actually do it in the test function itself. The reason being this might be used by many other tests as it is a utility function and will impact other tests too.

Reply:
I have added this "resource already exists" exception here in the exec function as the function was raising command failed irrespective of checking if the resource was present or not, but I get your point will change this utility and add the exception in the test.

Reply:
You can use ResourceAlreadyExists in the test function I meant.. Its better to be specific like you said. :)